packages/sftpgo/build.sh

```diff
@@ -0,0 +1,48 @@
+TERMUX_PKG_HOMEPAGE=https://sftpgo.com/
+TERMUX_PKG_DESCRIPTION="Full-featured and highly configurable SFTP, HTTP/S, FTP/S and WebDAV server"
+TERMUX_PKG_LICENSE="AGPL-V3"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="2.6.2"
+TERMUX_PKG_SRCURL=https://github.com/drakkan/sftpgo/releases/download/v$TERMUX_PKG_VERSION/sftpgo_v${TERMUX_PKG_VERSION}_src_with_deps.tar.xz
+TERMUX_PKG_SHA256=c90260b7b2901438bbd476eee9fd389af5af24113088a50284b2d170631b52ee
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE="latest-release-tag"
+TERMUX_PKG_HOSTBUILD=true
+
+termux_extract_src_archive() {
+	local file="$TERMUX_PKG_CACHEDIR/$(basename "${TERMUX_PKG_SRCURL}")"
+	mkdir -p "$TERMUX_PKG_SRCDIR"
+	tar xf "$file" -C "$TERMUX_PKG_SRCDIR" --strip-components=0
+}
+
+termux_step_host_build() {
+	termux_setup_golang
+
+	cd "$TERMUX_PKG_SRCDIR"
```

Comment:
Aren't we already in the source directory?
Or does `TERMUX_PKG_BUILD_IN_SRC=true` not apply to host builds?

Reply:
https://github.com/termux/termux-packages/blob/129940b929cee907e4c1af8d56cd10780b155da4/scripts/build/termux_step_handle_host_build.sh#L14-L15

Reply:
ah